### DISCUSSION TITLE
im getting an error when installing

### DISCUSSION THREAD

**Discussion by Elmax2022** [carson-katri/dream-textures#373] [Q&A] [answered]
ImportError:failed to load Pytorch C extensions:It appears that Pytorch has loaded the ‘torch/_C’ folder of the Pytorch repository rather than the C extensions which are expected in the ‘torch._C’ namespace.

It appears while generating the image in Dream

**Comment by carson-katri** [comment, marked as answer]
This is likely an issue with installing the incorrect version. If you are on Windows, make sure you install the `dream_textures-win32-cuda.7z` file.

**Reply by Elmax2022** [reply]
Thank you! It worked but can you tell me why its giving me errors during downloading CLIP and says ''Process closed unexpectedly''?

**Reply by carson-katri** [reply]
Please try this: https://github.com/carson-katri/dream-textures/issues/220#issuecomment-1264638666

**Reply by Elmax2022** [reply]
Thank u for ur answer. I recently bought new laptop and trying Dream via new one. But while generating it says 'Not enough memory, use lower resolution'. What can i do?

**Reply by serhiro89** [reply]
In the addon settings, when loading download Stable Diffusion v 2.1 (Recommended) nothing happens, does not start the download, selected all as required by the instructions, Why?
